bp_fe/src/include/bp_fe_itlb.vh

```diff
@@ -0,0 +1,66 @@
+/*                                                                         
+ * itlb.vh
+ * This file declares the internal structs of the BlackParrot Front
+ * End I-TLB module. For simplicity and flexibility, these structs are
+ * parameterized. Each struct declares its width separately to
+ * prevent pre-processor ordering issues. 
+*/
+
+`ifndef ITLB_VH
+`define ITLB_VH
+
+`ifndef BSG_DEFINES_V
+`define BSG_DEFINES_V
+`include "bsg_defines.v"
+`endif
+
+`ifndef BP_COMMON_FE_BE_IF
+`define BP_COMMON_FE_BE_IF
+`include "bp_common_fe_be_if.vh"
+`endif
+
+import itlb_pkg::*;
+/*
```

Comment:
add newline before the comment to separate it from the unrelated item above, and delete newline after comment that immediately precedes the `define. Do this for all comments before `defines
```suggestion

/*
```